UnityProject/Assets/Scripts/Weapons/Weapon.cs

```diff
@@ -51,10 +51,15 @@ public class Weapon : PickUpTrigger
 		/// </summary>
 		public string FireingSound;
 
-		/// <summary>
-		///     The amount of times per second this weapon can fire
-		/// </summary>
-		public double FireRate;
+        /// <summary>
+        ///     Checks if the weapon should spawn weapon casings
+        /// </summary>
+        public bool SpawnsCaseing = true; //be default treat weapons a spawning casing
+
+        /// <summary>
+        ///     The amount of times per second this weapon can fire
+        /// </summary>
+        public double FireRate;
```

Comment:
We enforce the use of Tabs instead of spaces, you continuesly use spaces, which we don't want.